tests/rptest/tests/upgrade_test.py

```diff
@@ -130,6 +130,13 @@ class UpgradeBackToBackTest(PreallocNodesTest):
     topics = (TopicSpec(partition_count=3, replication_factor=3), )
 
     def __init__(self, test_context):
+        if self.debug_mode:
+            self.MSG_SIZE = 10
+            self.RANDOM_READ_COUNT = 10
+            self.RANDOM_READ_PARALLEL = 1
+            self.CONSUMER_GROUP_READERS = 1
```

Comment:
These are instance variables, and there are also class variables with the same names initialized to different values above. Are they both used? This is somewhat confusing.

Reply:
These overwrite the class variables in debug mode.